app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -185,19 +186,31 @@ class TabTrayDialogFragment : AppCompatDialogFragment() {
             getString(R.string.snackbar_tab_closed)
         }
 
-        view?.tabLayout?.let {
-            viewLifecycleOwner.lifecycleScope.allowUndo(
-                it,
-                snackbarMessage,
-                getString(R.string.snackbar_deleted_undo),
-                {
-                    sessionManager.add(snapshot.session, isSelected, engineSessionState = state)
-                    tabTrayView.scrollToTab(snapshot.session.id)
-                },
-                operation = { },
-                elevation = ELEVATION,
-                anchorView = snackbarAnchor
-            )
+        val isLastOpenTab = sessionManager.sessions.size == 1
```

Comment:
So we only want to close the tabs tray if private and normal tabs are gone ?

Reply:
Still waiting on a UX answer for this, but I'm probably going to change it to be based on the active session type (private or normal).